platform/default/sqlite3.hpp

```diff
@@ -42,27 +48,23 @@ class Database {
 
     explicit operator bool() const;
 
-    static void errorLogCallback(void *arg, const int err, const char *msg);
     void setBusyTimeout(std::chrono::milliseconds);
     void exec(const std::string &sql);
     Statement prepare(const char *query);
 
-    int64_t lastInsertRowid() const;
```

Comment:
> so Qt will have the unexpected behavior if you have multiple statements and don't call numRowsAffected() / lastInsertId() immediately after executing each one?

Qt checks the state of each statement prior to calling these - a statement there is considered _active_ after it has been exec()'d. From my tests, though, two different statements, after being executed, do not deactivate each other, so I guess the behavior is unexpected indeed.